backend/jest.config.js

```diff
@@ -1,4 +1,6 @@
 module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
+  maxWorkers: 1,
+  testMatch: ['**/__tests__/**/*.ts?(x)', '**/?(*.)+(spec|test).ts?(x)'],
```

Comment:
The reason why it was not working before, was because jest would execute all tests in parallel, but we cannot afford to do this for our tests that interact with the database.

It was also not working because jest would run the `.js` test files for some reason, but we only want it to run `.ts` test files.